doc/development-workflow.md

```diff
@@ -0,0 +1,22 @@
+# Development workflow
+
+## PR author
+
+1. Open a PR with the changes you want to merge.
+
+2. Add the "Waiting on factcheck" label if the changes require feedback from someone other than developers.
+
+3. Move the associated Trello card from "In progress" to "Code review".
```

Comment:
Adding a `Waiting for factcheck` label whilst the PR is in code review and not currently being factchecked is not representative of the status of the PR though.